src/App.js

```diff
@@ -1,13 +1,71 @@
 import React from 'react';
 
 import './App.scss';
+import { GoodsList } from './components/GoodsList/GoodsList';
+import { getAll, get5First, getRedGoods } from './api/goods';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+class App extends React.Component {
+  state = {
+    renderList: [],
+  }
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+  handleAllgoods = () => (
+    getAll()
+      .then((allGoods) => {
+        this.setState({
+          renderList: allGoods,
+        });
+      })
+  );
+
+  handleFirstFiveGoods = () => (
+    get5First()
+      .then((first5goods) => {
+        this.setState({
+          renderList: first5goods,
+        });
+      })
+  )
+
+  handleRedGoods = () => (
+    getRedGoods()
+      .then((redGoods) => {
+        this.setState({
+          renderList: redGoods,
+        });
+      })
```

Comment:
Could you make 1 function that accepts a callback as a parameter, instead of these 3?